server/src/services/RedisClient.js

```diff
@@ -0,0 +1,42 @@
+const { promisify } = require('util');
+
+class RedisClient {
+    constructor(redisClient) {
+        this.keyExpiresInMinutes = 0;
+        this.redis = redisClient;
+        ['json_get', 'json_set', 'hgetall', 'hset', 'hget', 'hdel', 'hincrby', 'scan'].forEach(
+            method => (this.redis[method] = promisify(this.redis[method]))
+        );
+    }
+
+    async eachScan(pattern) {
+        let matchingKeysCount = 0;
+        let keys = [];
+
+        const recursiveScan = async (cursor = 0) => {
```

Comment:
shouldnt be `cursor = '0'` here? because in you have `if (cursor === '0')`

Reply:
done